docs/community/index.md

```diff
@@ -0,0 +1,61 @@
+# Community Calls and Events
```

Comment:
```suggestion
# Calls and Events
```

It may be my bias, but I think this phrasing could be confused with the more broad [Jupyter Community Calls](https://discourse.jupyter.org/t/jupyter-community-calls/668/) since I already have people asking me about other calls that use this language.